examples/kibana-sample-data/docker-compose.yml

```diff
@@ -50,11 +50,11 @@ services:
   kibana:
     image: docker.elastic.co/kibana/kibana:8.11.1
     environment:
-      ELASTICSEARCH_HOSTS: '["http://quesma:8080"]'
+      ELASTICSEARCH_HOSTS: '["http://10.16.10.6:8080"]'
```

Comment:
If you like custom ip addresses, you may add another folder in `examples/` such as `examples/dev-...` and then add it to `.gitignore`.